### ISSUE TITLE
Can the shortcut be customized?

### ISSUE THREAD

**Issue by tnguyen14** [aigarsdz/olden#10]
It would be great if the app shortcut can be changed from `Alt+Space` to something else.

**Comment by aigarsdz** [comment]
I agree. That's one of the features I've planned for the next version, but I've been distracted with other projects lately, so the development of the new Olden, which is going to be a pretty significant update, is very slow.
